src/stories/TextInput.stories.js

```diff
@@ -57,6 +57,70 @@ Placeholder.args = {
     placeholder: 'My placeholder text',
 };
 
+const AutoGrow = Template.bind({});
+AutoGrow.storyName = 'Autogrow input';
+AutoGrow.args = {
+    label: 'Autogrow input',
```

Comment:
Let's standardize the case-convention for the labels: `AutoGrow Input` since the other labels seem to capitalize everything.

Reply:
It seems older ones have first as only capital letter so I am going with that following the page titles in the app.

Reply:
It looks like there are still two main conventions for labels: 

1. `Input with xyz` - e.g. `Input with default value` 
2. `xyz input`  - e.g. `Prefixed input`

I'm fine with either options but i do think we should pick one

Reply:
Updated most of them. Left few whose name makes more sense. But I would say it does not matter much. The main perspective should be that they should clearly show the proper example of that component/varient